### PR TITLE
General Edits

### PR DESCRIPTION
**Pull request by Catografix**
Some questions about custom script syntax, see line notes

### CODE REVIEW

File: articles/connections/social/oauth2.md [hunk 1]
```diff
@@ -35,39 +36,42 @@ curl -H "Content-Type: application/json" -H 'Authorization: Bearer {YOUR_GLOBAL_
...
 
-The script will have the following signature:
+### Custom script
 
-```
-function(ctx,access_token,callback){
+A custom script will have the following signature:
+
+```js
+function(ctx, access_token, callback){
```
Comment (Catografix) [inline articles/connections/social/oauth2.md]: In all the examples at https://auth0.com/docs/oauth2-examples, the syntax is "function(accessToken, ctx, cb)" - that is accessToken is the first parameter, not the second.

File: articles/connections/social/oauth2.md [hunk 2]
```diff
@@ -35,39 +36,42 @@ curl -H "Content-Type: application/json" -H 'Authorization: Bearer {YOUR_GLOBAL_
...
+The `access_token` parameter can be used for authenticating requests to the provider's API.
 
-For example, retrieving the user profile from __GitHub__ would look like:
+**NOTE:** We recommend using the field names from the [normalized profile](/user-profile#normalized-user-profile).
 
-```
+For example, the following code will retrieve the user profile from **GitHub**:
+
+```js
 function(ctx, access_token, callback) {
```
Comment (Catografix) [inline articles/connections/social/oauth2.md]: In all the examples at https://auth0.com/docs/oauth2-examples, the syntax is `function(accessToken, ctx, cb)` - that is accessToken is the first parameter, not the second.

File: articles/connections/social/oauth2.md [hunk 3]
```diff
@@ -35,39 +36,42 @@ curl -H "Content-Type: application/json" -H 'Authorization: Bearer {YOUR_GLOBAL_
...
-The `access_token` parameter will often be used for authenticating requests to the provider's API.
+The `access_token` parameter can be used for authenticating requests to the provider's API.
 
-For example, retrieving the user profile from __GitHub__ would look like:
+**NOTE:** We recommend using the field names from the [normalized profile](/user-profile#normalized-user-profile).
 
-```
+For example, the following code will retrieve the user profile from **GitHub**:
+
+```js
```
Comment (Catografix) [inline articles/connections/social/oauth2.md:74]: In all the examples at https://auth0.com/docs/oauth2-examples, the syntax is "function(accessToken, ctx, cb)" - that is accessToken is the first parameter, not the second.

File: articles/connections/social/oauth2.md [hunk 4]
```diff
@@ -35,39 +36,42 @@ curl -H "Content-Type: application/json" -H 'Authorization: Bearer {YOUR_GLOBAL_
...
+* **tokenURL**: the URL Auth0 will use to exchange the `code` for an `access_token`. Usually similar to `https://your.oauth2.server/oauth2/token`.
+* **scope**: the scope parameters that you want to request consent for (e.g. `profile`, etc.).
+* **fetchUserProfile**: a custom script that returns a JSON object with user info.
 
-The script will have the following signature:
+### Custom script
 
-```
-function(ctx,access_token,callback){
+A custom script will have the following signature:
```
Comment (Catografix) [inline articles/connections/social/oauth2.md]: This example doesnt seem to be about the custom script, but about the format of the user profile.

File: articles/connections/social/oauth2.md (hunk 1 above)
Comment (ntotten) [inline articles/connections/social/oauth2.md]: This is wrong. it should be like the others.

Comment (ntotten) [inline articles/connections/social/oauth2.md]: All of these functions should be in the format:

```
function(access_token,  ctx, callback){
}
```

File: articles/connections/social/oauth2.md (hunk 4 above)
Comment (ntotten) [inline articles/connections/social/oauth2.md]: The wording might be bad. It is a "custom script" to get the user profile.

**Pull request merged**